smallrye-reactive-messaging-aws-sns/src/test/java/io/smallrye/reactive/messaging/aws/sns/MapBasedConfig.java

```diff
@@ -25,6 +25,7 @@ public MapBasedConfig(Map<String, Object> map) {
     public static void clear() {
         File out = new File("target/test-classes/META-INF/microprofile-config.properties");
         if (out.delete()) {
```

Comment:
Should this be `out.exists()`?

Reply:
There are several places where this if check is made in the code.

In some places the if was removed. in other it still says `out.delete()` and in another, it was changed to `out.exists()`. What should it be?

Reply:
Same, it should have been `isFile`. Fixing.